.github/workflows/test-application.yaml

```diff
@@ -22,9 +22,15 @@ jobs:
                 include:
                     - php-version: '8.0'
                       dependencies: 'lowest'
+                      phpunit-config: 'phpunit-9.xml.dist'
                     - php-version: '8.0'
+                      phpunit-config: 'phpunit-9.xml.dist'
                     - php-version: '8.1'
+                    - php-version: '8.2'
+                    - php-version: '8.3'
                       coverage: 'true'
+                    - php-version: '8.4'
+                      composer-flags: '--ignore-platform-reqs'
```

Comment:
can be removed as soon as prophecy supports 8.4 officially: https://github.com/phpspec/prophecy/blob/f9e07be0992e7bf1cad210829055b99318df142f/composer.json#L21 but mostly it takes some time.